liberty-ls/src/main/java/io/openliberty/tools/langserver/utils/ServerPropertyValues.java

```diff
@@ -23,6 +23,7 @@
 public class ServerPropertyValues {
     private final static List<String> LOGGING_SOURCE_VALUES = Arrays.asList("message", "trace", "accessLog", "ffdc", "audit");
     private final static List<String> BOOLEAN_VALUES = Arrays.asList("true", "false");
+    private final static List<String> BOOLEAN_VALUES_REVERSE = Arrays.asList("false", "true");
```

Comment:
Would make more sense to call these BOOLEAN_VALUES_DEFAULT_TRUE and BOOLEAN_VALUES_DEFAULT_FALSE.